tests/e2escenarios/e2e_source_test.go

```diff
@@ -29,118 +30,127 @@ var _ = Describe("odo source e2e tests", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
-	Context("odo component creation", func() {
+	Context("odo component creation from source", func() {
+		JustBeforeEach(func() {
+			if runtime.GOARCH == "s390x" {
+				Skip("Skipping test because there is no supported builder image.")
+			}
+		})
 
-		It("Should be able to deploy a python source application", func() {
```

Comment:
I am confused 🙄 why are we changing the existing component specific test

Reply:
@prietyc123 I didn't change it. It's because I picked the skipped testcase and the line number for this test changed.

Reply:
ooops... My bad.

Reply:
@prietyc123 Test are just re arranged